src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceKindFunctions.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util
+
+import java.time.LocalDate
+
+fun determineReleaseDateKind(prrd: LocalDate?, crd: LocalDate?): LicenceKind = when {
+  // hdcad?.isTodayOrInTheFuture() == true -> LicenceKind.HDC
```

Comment:
Commented out to ensure HDC cases don't appear unexpectedly in CVL.
Note: This logic for determining HDC cases has NOT been confirmed, and should be before uncommenting. It has been left as an example of where the change will need to be added in future.